scripts/codecov_upload.sh

```diff
@@ -1,9 +1,18 @@
 #!/usr/bin/env bash
 
-# Script used to collect and upload test coverage (mostly by travis).
-# Usage ./test_coverage_upload.sh [log_file]
+# Script used to collect and upload test coverage.
 
 set -o pipefail
 
 # We try to upload whatever we have:
-bash <(curl -s https://codecov.io/bash) -f ./covdir/all.coverprofile -cF all
+mkdir -p bin
+curl -sf -o ./bin/codecov.sh https://codecov.io/bash
+
+bash ./bin/codecov.sh -f "${COVERDIR}/all.coverprofile" \
+  -cF all \
+  -C "${PULL_PULL_SHA}" \
+  -r "${REPO_OWNER}/${REPO_NAME}" \
+  -P "${PULL_NUMBER}" \
+  -b "${BUILD_ID}" \
+  -B "${PULL_BASE_REF}" \
+  -N "${PULL_BASE_SHA}"
```

Comment:
This seems to be an unrelated enhancement. It would be better to do it separately. But I won't insist on it.

Reply:
It's not unrelated because we will publish the reports from the Prow job; we need to specify these variables/arguments. In the GitHub actions environment, CodeCov would assume these values.